### ISSUE TITLE
Exception in XUnit2ScenarioOutlineExampleMatcher

### ISSUE THREAD

**Issue by magicmonty** [picklesdoc/pickles#548]
Given the following feature:

```gherkin
Feature: Test feature

Scenario Outline: Test outline
  Failing Test case

  Given "A" contains a line with "<link>"
  When I click onto "<link>"
  Then the link will link to "<result>"

  Examples:
  | link | result | remark |
  | FOO  | BAR    | BAZ    |
  | XYZ  |        |        |
```
and the following TestResultsFile:

```xml
<?xml version="1.0" encoding="utf-8"?>
<assemblies timestamp="10/01/2018 16:15:09">
  <assembly name="Requirements.DLL" environment="32-bit .NET 4.0.30319.42000 [collection-per-class, non-parallel]" test-framework="xUnit.net 2.4.0.4049" run-date="2018-10-01" run-time="16:15:09" config-file="Requirements.dll.config" total="5" passed="0" failed="0" skipped="5" time="0.366" errors="0">
    <errors />
    <collection total="1" passed="0" failed="0" skipped="1" name="Test collection for Requirements.TestFeature" time="0.000">
      <test name="Test outline" type="Requirements.TestFeature" method="TestOutline" time="0" result="Skip">
        <traits>
          <trait name="FeatureTitle" value="Test feature" />
          <trait name="Description" value="Test outline" />
          <trait name="Category" value="NotTested" />
          <trait name="Category" value="longrunning" />
        </traits>
        <reason><![CDATA[Ignored]]></reason>
      </test>
    </collection>
  </assembly>
</assemblies>
```

There is a System.ArgumentException with the message "Die Zeichenfolge kann keine Länge von 0 (null) haben.\r\nParametername: oldValue" (in English "The string cannot hav a length of 0 (null)....") in `XUnit2ScenarioOutlineExampleMatcher.ScenarioOutlineExampleIsMatch`

This was introduced in PR #509 

I have a fix and send a PR

**Comment by dirkrombauts** [comment]
Cool, thank you! I welcome your PR and will look at it as soon as I can!

**Comment by dirkrombauts** [comment]
Released in version [2.20.0](https://github.com/picklesdoc/pickles/releases/tag/v2.20.0).
